### PR TITLE
[SW-1249] Feat: Allow to access app browser on dev

### PR DESCRIPTION
**Pull request by peronczyk**

### CODE REVIEW

**Comment by martinkaintas** [comment]
I suppose we want all the buttons that redirect to the in-app-browser to be shown as well
Those are all the places where they appear:

https://github.com/aeternity/superhero-wallet/blob/d6f23ac531fe947c1daae9b14d2a262c4bb6b755/src/popup/components/Header.vue#L50

https://github.com/aeternity/superhero-wallet/blob/d6f23ac531fe947c1daae9b14d2a262c4bb6b755/src/popup/pages/Dashboard.vue#L16

https://github.com/aeternity/superhero-wallet/blob/d6f23ac531fe947c1daae9b14d2a262c4bb6b755/src/protocols/aeternity/views/AccountDetails.vue#L18

**Comment by peronczyk** [comment]
> I suppose we want all the buttons that redirect to the in-app-browser to be shown as well Those are all the places where they appear:
> 
> https://github.com/aeternity/superhero-wallet/blob/d6f23ac531fe947c1daae9b14d2a262c4bb6b755/src/popup/components/Header.vue#L50
> 
> https://github.com/aeternity/superhero-wallet/blob/d6f23ac531fe947c1daae9b14d2a262c4bb6b755/src/popup/pages/Dashboard.vue#L16
> 
> https://github.com/aeternity/superhero-wallet/blob/d6f23ac531fe947c1daae9b14d2a262c4bb6b755/src/protocols/aeternity/views/AccountDetails.vue#L18

Totally makes sense. Please see now @martinkaintas .

File: src/popup/pages/Dashboard.vue [hunk 1]
```diff
@@ -13,13 +13,14 @@
...
       <DashboardCard
-        v-if="IS_CORDOVA"
+        v-if="IS_CORDOVA || UNFINISHED_FEATURES"
         :title="$t('dashboard.daeppBrowserCard.title')"
         :description="$t('dashboard.daeppBrowserCard.description')"
         :btn-text="$t('dashboard.daeppBrowserCard.button')"
         :background="daeppBrowserBackground"
         :icon="GlobeIcon"
         :to="{ name: ROUTE_APPS_BROWSER }"
+        :card-id="DASHBOARD_CARD_ID.daeppBrowser"
```
Comment (martinkaintas) [inline src/popup/pages/Dashboard.vue:23]: I believe this was intentionally left out, because we don't want this card to be dismissable. @ZGiovas?

**Review by martinkaintas — APPROVED** [review]

File: src/popup/pages/Dashboard.vue (hunk 1 above)
Comment (ZGiovas) [reply src/popup/pages/Dashboard.vue:23]: yes @martinkaintas is right

Comment (peronczyk) [reply src/popup/pages/Dashboard.vue:23]: OK, I consulted this also with Paolo. I suggest to leave this line but to pass `null` from the constant. Here's the change with comment: https://github.com/aeternity/superhero-wallet/pull/2305/files#diff-9b9a49cd15349e68109f9c49f06bbda53dc9f38b4885771124ee55a2cf0dbf27R367
What do you think?

**Comment by martinkaintas** [comment]
lgtm

File: src/popup/pages/Dashboard.vue (hunk 1 above)
Comment (martinkaintas) [reply src/popup/pages/Dashboard.vue:23]: Looks good to me

**Review by ZGiovas — APPROVED** [review]

**Pull request merged**